src/usePlaidLink.ts

```diff
@@ -62,7 +62,7 @@ export const usePlaidLink = (options: PlaidLinkOptions) => {
 
   return {
     error,
-    ready: !loading || iframeLoaded,
+    ready: (!loading || iframeLoaded) && !!plaid,
```

Comment:
is there a reason you used `!!`?

```suggestion
    ready: plaid != null && (!loading || iframeLoaded),
```